### PR TITLE
Update UG for Service functionality

### LINKED ISSUE #57: Update UG for Service
Add Service functionality related commands to User Guide

### PR DESCRIPTION
**Pull request by hanstirtaputra**
Added commands for services functionality to the UG in accordance with the structure. A screenshot of the example outcome has not yet been added.

Let's include a numbering system for each component in our UG, for example **Features** would be under section number 4. Hence each individual feature should be should be labelled as 4.1, 4.2, 4.3 ... and each command associated with that feature be labelled with 4.1.1, 4.1.2, 4.1.3 ...

### CODE REVIEW

**Review by khoongwk — APPROVED** [review]
LGTM. The numbering of features is good and we should stick to it.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -113,6 +112,151 @@ Example:
...
+Let's say you have a service with the following information you want to add into GrAB3. You can follow these instructions.
+
+| Service | |
+|---------|--------- |
+|`TITLE`| Lash Lift |
+|`DURATION`| 0.5 |
+|`PRICE`| 38 |
+
+Adding the above service:
+1. Type `addsvc t/Lash Lift d/0.5 p/38' into the *Command Box*.
```
Comment (khoongwk) [inline docs/UserGuide.md]: I think you're missing a quotation mark at the end here.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -113,6 +112,151 @@ Example:
...
+|`TITLE`| The title of the service you are providing. It must be alphanumeric words not more than 50 characters long.|E.g `Lash Lift`
+|`DURATION`| The duration of the service in hours. <br> <br> It be in half hour intervals.| E.g `1.5`
+|`PRICE`| The revenue received from the service. <br> <br> It must be in dollars.| E.g `5.50`
+
+#### 4.1.2 Add a new service: `addsvc`
+
+You can use this command to add a new service GrAB3.
+
+Format: `addsvc t/TITLE d/DURATION p/PRICE`
+
```
Comment (khoongwk) [inline docs/UserGuide.md:133]: I think we need a additional section below every "Format" to tell the user to reference the command parameters. For example:
ℹ️ | Refer to Section 4.1.1, “Service Management Command Parameters” for more detail about each parameter.
-- | --

Comment (khoongwk) [reply docs/UserGuide.md:133]: And replicate this 'success' for the other parts till 4.1.7 😄

**Review by khoongwk — CHANGES_REQUESTED** [review]
Thought about some improvements while doing my own UG 😅

File: docs/UserGuide.md (hunk 2 above)
Comment (khoongwk) [reply docs/UserGuide.md:133]: Can refer to @galvinleow 's PR for this ^.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -113,6 +112,151 @@ Example:
...
 {Example outcome screenshot}
 
+## 4.1 Services Management
+
+This feature allows you to manage the services that your home-based beauty salon provides. You can record the 
+following information about your services: `TITLE`, `DURATION` and `PRICE`.
+
+### 4.1.1 Service Management Command Parameters
+
+Before you dive into using the feature, you may want to have a look at the common parameter used in this feature.
```
Comment (yanlynnnnn) [inline docs/UserGuide.md]: Should it be 'parameters' instead of 'parameter'?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -113,6 +112,151 @@ Example:
...
+The table below shows a list of command parameters that will be used in this feature.
+
+| Parameter Name | Description | Example
+|---------|---------|---------
+|`TITLE`| The title of the service you are providing. It must be alphanumeric words not more than 50 characters long.|E.g `Lash Lift`
+|`DURATION`| The duration of the service in hours. <br> <br> It be in half hour intervals.| E.g `1.5`
+|`PRICE`| The revenue received from the service. <br> <br> It must be in dollars.| E.g `5.50`
```
Comment (yanlynnnnn) [inline docs/UserGuide.md:125]: Should SERVICE_CODE be added here?

**Review by yanlynnnnn — COMMENTED** [review]
I think you might have missed out the command summary section at the bottom, but the rest looks good.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -113,6 +112,151 @@ Example:
 
 {Example outcome screenshot}
 
+## 4.1 Services Management
+
+This feature allows you to manage the services that your home-based beauty salon provides. You can record the 
+following information about your services: `TITLE`, `DURATION` and `PRICE`.
+
+### 4.1.1 Service Management Command Parameters
```
Comment (khoongwk) [inline docs/UserGuide.md]: 4.1.1 should be on the same level as 4.1.2. Should we reduce the heading level for the commands below from #### to ### to match this?

**Review by khoongwk — CHANGES_REQUESTED** [review]
Question regarding heading levels for command parameters

**Comment by hanstirtaputra** [comment]
Thanks for the comments @khoongwk @yanlynnnnn , I have made the changes as requested!

**Comment by hanstirtaputra** [comment]
Thanks for the comments @khoongwk @yanlynnnnn, I have made the changes as requested!

**Review by khoongwk — APPROVED** [review]
LGTM 😄

**Pull request merged**